build/build.ps1

```diff
@@ -7,11 +7,37 @@ $ErrorActionPreference = 'Stop'
 $all_ok = $True
 Write-Host "Assembly version: $Env:ASSEMBLY_VERSION"
 
-choco install llvm --version=11.1.0 --allow-downgrade
-if (!(Get-Command clang -ErrorAction SilentlyContinue) -and (choco find --idonly -l llvm) -contains "llvm") {
-    # For some reason, adding to the path does not work on our build servers, even after calling refreshenv.
-    # LLVM was installed by Chocolatey, so add the install location to the path.
-    $env:PATH += ";$($env:SystemDrive)\Program Files\LLVM\bin"
+if (($IsWindows) -or ((Test-Path Env:/AGENT_OS) -and ($Env:AGENT_OS.StartsWith("Win")))) {
+    if (!(Get-Command clang        -ErrorAction SilentlyContinue) -or `
+        !(Get-Command clang-format -ErrorAction SilentlyContinue) -or `
+        (Test-Path Env:/AGENT_OS)) {
+        choco install llvm --version=13.0.0 --allow-downgrade
```

Comment:
I'm worried that prepending LLVM to the path effects the private lookup path, but I'm not 100% sure.

Reply:
It's an interesting question... but I think it's mitigated somewhat by how we are trying to ensure the compiler QIR Generation step explicitly loads the binaries that we build, even if the environment has another LLVM in it. So it shouldn't interfere with the generation step, and if it does then that would be something we should fix in the compiler itself.

Reply:
I think we can avoid installing clang entirely in the script, so this has been removed.